### PR TITLE
AnkitaS_ProgrammingTest_2

### PR DESCRIPTION
**Pull request by ankitasambhare12**
### Assignment Number :1234:
Example: Programmingtest:-2

### Assignment Description :notebook:
Programming test - 2 : 02 Feb'2024
Print last consonant from the string

input : aakansha
output : h
Print last even number from a String

input : te3ch4nocred8its
output : 8

### Evidence :file_folder:
image
image

### Checklist before requesting a review :ballot_box_with_check:

- [x] I have performed a self-review of my code
- [x] My changes generate no compile time errors

### CODE REVIEW

File: src/ankita_s/LastConsonantInString.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package ankita_s;
+
+public class LastConsonantInString {
+	char conso;
+
+	void printLastConsonantInString(String str) {
+
+		for (int index = 0; index < str.length(); index++) {
+			char ch = str.charAt(index);
+			if (ch != 'a' && ch != 'e' && ch != 'i' && ch != 'o' && ch != 'u') {
```
Comment (MargiVegada) [inline src/ankita_s/LastConsonantInString.java:10]: You can use OR condition

File: src/ankita_s/LastEevenNoInString.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+			int temp1=Character.getNumericValue(ch);
+			if(temp==true && temp1%2==0)
+			{
+				temp2=temp1;
+				}
+			}
+	   System.out.println("Last even no is:- " +temp2);
+	}
+	
+public static void main(String[]ars)
```
Comment (MargiVegada) [inline src/ankita_s/LastEevenNoInString.java:20]: 1 tab space needed

File: src/ankita_s/LastEevenNoInString.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+		{
+			char ch = Str.charAt(index);
+			boolean temp = Character.isDigit(ch);
+			int temp1=Character.getNumericValue(ch);
+			if(temp==true && temp1%2==0)
+			{
+				temp2=temp1;
+				}
+			}
+	   System.out.println("Last even no is:- " +temp2);
```
Comment (MargiVegada) [inline src/ankita_s/LastEevenNoInString.java:17]: All statements are not aligned

**Review by MargiVegada — APPROVED** [review]
Format your code before pushing

File: src/ankita_s/LastEevenNoInString.java (hunk 3 above)
Comment (ankitasambhare12) [reply src/ankita_s/LastEevenNoInString.java]: Formatted the code

**Pull request merged**